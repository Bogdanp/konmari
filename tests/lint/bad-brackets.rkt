#lang racket/base

(let ((x 1)
      [y 2]
      (z 3))
  x)
