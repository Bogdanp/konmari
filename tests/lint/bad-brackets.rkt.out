/bad-brackets.rkt:3:7:warning:bindings within a let should be surrounded by square brackets
/bad-brackets.rkt:5:7:warning:bindings within a let should be surrounded by square brackets
/bad-brackets.rkt:5:8:warning:identifier 'z' is never used
/bad-brackets.rkt:4:8:warning:identifier 'y' is never used
